src/expressions/loop-expr.md

```diff
@@ -226,6 +241,15 @@ Example:
 
 A `break` expression is only permitted in the body of a loop, and has one of the forms `break`, `break 'label` or ([see below](#break-and-loop-values)) `break EXPR` or `break 'label EXPR`.
 
+## Label expressions
+
+> **<sup>Syntax</sup>**\
+> _LabelBreakExpression_ :\
+> &nbsp;&nbsp; [LIFETIME_OR_LABEL] [_BlockExpression_]
```

Comment:
I might choose a different name for this, since I think "label break expression" is a little too similar _BreakExpression_. To me, it makes it sound like `'label break expr`. What do you think of renaming it to something like _LabelBlockExpression_?

Also I think it would be good to add this to the _LoopExpression_ definition at the top of the page and drop the `LIFETIME_OR_LABEL`. That way it can share the definition for _LoopLabel_ (and, btw, this is missing the `:`).

So, this might be something like:

```suggestion
> _LabelBlockExpression_ :\
> &nbsp;&nbsp; [_BlockExpression_]
```